AirportTicketNookingTest/BookingShould.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Airport_Ticket_Booking;
+using AutoFixture;
+using FluentAssertions;
+using Xunit;
+using static Airport_Ticket_Booking.Common;
+
+namespace AirportTicketNookingTest
+{
+    public class BookingShould
+    {
+        private readonly Fixture _fixture;
+        private readonly FlightsManager _flightManager;
+
+        public BookingShould()
+        {
+            _fixture = new Fixture();
+            _flightManager = FlightsManager.Instance();
```

Comment:
testing singleton pattern is a bit tricky, why?

In order to test the singleton pattern you may consider something like adding a method to reset the instance you created between tests, something like: 
```    
public static void ResetInstance()
    {
        _instance = null;
    }
```